client/my-sites/scan/style.scss

```diff
@@ -46,25 +55,35 @@
 	}
 }
 
-header.scan__header.is-left-align {
+.scan__header {
 	color: var( --studio-black );
-	font-size: 28px;
+	font-size: rem( 28px );
```

Comment:
Out of curiosity, what does it do wrapping the `font-size` with `rem`?

Reply:
@rcanepa it outputs two `font-size` properties:
- One in rem units for the [browsers that support it](https://caniuse.com/#feat=rem).
- A fallback in pixels for older browsers.

Reply:
Oh, that's really cool!

Reply:
Neat! Assume that's a SASS thing?

Reply:
Whoops — ignore me. The way I explained it it's how it is (was?) usually implemented, but apparently we don't have the pixel fallback in Calypso, so `rem()` just converts a pixel value to a rem value using  `16px` as the base font size:

https://github.com/Automattic/wp-calypso/blob/master/client/assets/stylesheets/shared/_typography.scss#L27-L39